casaos-healthcheck/run.sh

```
@@ -381,6 +381,325 @@ check_dmesg_errors() {
+        if [[ $ntp_status == *"yes"* ]]; then
+            print_color "0;32" "${CHECK_MARK} NTP is synchronized"
+        else
+            print_color "0;31" "${CROSS_MARK} NTP is not synchronized"
+        fi
+        
+        local time_status=$(timedatectl status --no-pager)
+        echo "System time status:"
+        echo "$time_status"
+    else
+        if command -v ntpq &>/dev/null; then
+            local ntp_peers=$(ntpq -p)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)